ipfs-pinning-service.yaml

```
@@ -392,15 +417,25 @@ components:
 
   parameters:
 
-    skip:
-      description: number of items to skip
-      name: skip
+    before:
+      description: remove results created (queued) after provided timestamp
```

`remove` makes it sound like deletes are happening.
```suggestion
      description: return results created (queued) before provided timestamp
```